### ISSUE TITLE
Investigate new progressbar behaviour for pipe to file

### ISSUE THREAD

**Issue by Lestropie** [MRtrix3/mrtrix3#1744]
Presumably related to #1707 / #1586.

Have noticed lately in the CI outputs on `dev`-based branches that the progressbars when `stderr` is a file (i.e. "`[=====]`") are enormous and sometimes take up a substantial fraction of the terminal output. It's possible that in the process of these changes, the mechanisms for limiting the length of these non-updateable progress bars has been bypassed.

**Comment by thijsdhollander** [comment]
Yes, I can confirm this was at least already the case when I reported #1710 .  I removed the progress bar related output manually there, as it was causing excessive clutter indeed.
